src/EPR.SubsidiaryBulkUpload.Application/Models/OrganisationModel.cs

```diff
@@ -26,4 +26,8 @@ public class OrganisationModel
     public Nation? Nation { get; set; }
 
     public string? SubsidiaryOrganisationId { get; set; }
+
+    public string? CHName { get; set; }
```

Comment:
Use the full name - CompaniesHouseName or CompaniesHouseCompanyName or NameOnCompaniesHouse

Reply:
changed to one more readable.